src/Microsoft.Security.Utilities.Core/PreciselyClassifiedSecurityKeys/IdentifiableKey.cs

```diff
@@ -25,6 +25,9 @@ public IdentifiableKey()
 
         public abstract IEnumerable<ulong> ChecksumSeeds { get; }
 
+        IEnumerable<HighPerformancePattern> IHighPerformanceScannableKey.HighPerformancePatterns => HighPerformancePatterns;
+        private protected abstract IEnumerable<HighPerformancePattern> HighPerformancePatterns { get; }
```

Comment:
>private protected [](http://example.com/codeflow?start=8&length=17)

what the heck? FamAndAssembly now supported??

Reply:
Yes.